<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemWidget</class>
 <widget class="QWidget" name="ItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,2">
   <item row="0" column="0">
    <widget class="QLabel" name="item_Label">
     <property name="text">
      <string>Article : </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="manufacturerCode_LineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="partNumber_LineEdit"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="itemName_LineEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>manufacturerCode_LineEdit</tabstop>
  <tabstop>partNumber_LineEdit</tabstop>
  <tabstop>itemName_LineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
